### ISSUE TITLE
use lsst.obs.lsst.imsim.ImsimMapper from obs_lsst package 

### ISSUE THREAD

**Issue by jchiang87** [LSSTDESC/imSim#177]
Once `obs_lsstCam` is renamed to `obs_lsst` and integrated into `lsst_distrib`, we need to update the code in `python/desc/imsim/camera_info.py` to use `lsst.obs.lsst.imsim.ImsimMapper` so that the imSim readout parameters (i.e., crosstalk, read noise, gains) are used.

**Comment by cwwalter** [comment]
@heather999 and @danielsf Can you summarize what the status of the camera package transition is and when we need to make this change that Jim tagged?

**Comment by danielsf** [comment]
I am not sure that any progress has been made on incorporating `obs_lsst` into `lsst_distrib`.

The relevant DM ticket is

https://jira.lsstcorp.org/browse/DM-16374?jql=text%20~%20%22obs_lsstCam%20RFC%22

**Comment by heather999** [comment]
The underlying package which is still referred to as "obs_lsstCam" in GitHub, is already renamed to "obs_lsst". So effectively, you have to do a "setup obs_lsst" to setup the package these days.  DM has started to make the transition, by turning on some of their protections on this repo: https://lsstc.slack.com/archives/CBE964PR8/p1541014023020800
In our conversations with Leanne, the plan is that obs_lsst will be part of the upcoming v17 DMstack release and that is due to come out this month.  So if that all holds up, this transition should be "soon".

**Comment by cwwalter** [comment]
Thanks both.  Do we not make the change Jim refers to above (using lsst.obs.lsst.imsim.ImsimMapper) until that happens?

**Comment by cwwalter** [comment]
Is using  lsst.obs.lsst.imsim.ImsimMapper something we can only do after the obs_lsst transition? Or does it also work with obs_lsstCam?

**Comment by heather999** [comment]
We don't have to wait for the transition to obs_lsst - that name change to lsst.obs.lsst.imsim.ImsimMapper has been available in obs_lsstCam both in master and along our dc2/run1.2-v2 branch and in the w.2018.39-run1.2 tag). This is compatible with DMstack w_2018_39.   The question might be, what version of obs_lsstCam/DMstack weekly are you guys planning to use for Run2.1i?

**Comment by cwwalter** [comment]
This issue used for both transitioning to new package and using the new mapper.
